docs/contributing/platform_api/exceptions.rst

```diff
@@ -0,0 +1,16 @@
+pulp.exceptions
+===============
+
+All exceptions documented here should be imported directly from the ``pulp.exceptions`` namespace.
```

Comment:
This convention exists in a few modules, including some over in pulp.app. I'm sticking with it for this PR, but think that this message should actually live in the docstring of `pulp.exceptions` itself, so you see it when editing code and don't have to go to the rendered docs to know the correct way to import this stuff.